zerver/openapi/zulip.yaml

```diff
@@ -9142,6 +9142,47 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+  /mobile_push/test_notification:
+    post:
+      operationId: test-notify
+      summary: Sends a test notifications to the user's mobile device(s)
+      tags: ["mobile"]
+      description: |
+        This endpoint allows a user to trigger a test push notification to their
+        selected mobile devices, or all their mobile devices.
+
+        **Changes**: New in Zulip 8.0 (feature level 217).
+      parameters:
+        - name: token
+          in: query
+          description: |
+            The push token for the device to send the test notification to.
```

Comment:
"The push token for the device to which to send the test notification."

Or some other rewording. The thing with the existing version is that its syntax is ambiguous — and at least for me, my first reading is that it means the *device* will send the test notification to this push token. Where what's meant is that an unspecified subject (implicitly the server) will send the test notification to the device associated with this push token.